procedures/katello/release.md.erb

```diff
@@ -14,42 +14,54 @@
 - [ ] Do Cherry-picks: Clone [tool_belt](https://github.com/theforeman/tool_belt)
   - [ ] If any minor versions of Katello have been added to Redmine since the last cherry-pick, make sure to include them in `prior_releases` in configs/katello/<%= short_version %>.yaml
   - [ ] Run `./tools.rb setup-environment configs/katello/<%= short_version %>.yaml`
-  - [ ] Run `GITHUB_ACCESS_TOKEN=<secret> ./tools.rb cherry-picks --version <%= full_version %> configs/katello/<%= short_version %>.yaml`
+  - [ ] Run `GITHUB_ACCESS_TOKEN=<secret> PROJECT=katello VERSION=<%= short_version %> ./tools.rb cherry-picks configs/katello/<%= short_version %>.yaml`
```

Comment:
This doesn't look correct because tool_belt doesn't respect `PROJECT` and `VERSION`.